ui/Makefile

```diff
@@ -46,8 +46,9 @@ CSS_TARGET              := build/app.css
 CSS_DEBUG_TARGET        := build/app_debug.css
 REMOTE_DEPS             := npm.installed bower.installed tsd.installed
 INDEX                   := index.html
-GOBINDATA_DEPS          := $(TYPESCRIPT_TARGET) $(CSS_TARGET)
-GOBINDATA_DEBUG_DEPS    := $(TYPESCRIPT_TARGET) $(CSS_DEBUG_TARGET)
+GOBINDATA_FONTS         := ./fonts/*
```

Comment:
`GOBINDATA_FONTS` should be added to `GOBINDATA_SOURCES`. It doesn't make sense to depend on it because it is neither fully enumerated nor does it have targets in case it's missing